resources/prometheus/prometheus-rules.yaml

```diff
@@ -236,13 +236,16 @@ spec:
     - name: rhacs-probe
       rules:
         - alert: RHACSProbeRunFailed
-          expr: acs_probe_last_failure_timestamp > 0 and acs_probe_last_failure_timestamp >= acs_probe_last_success_timestamp
+          expr: |
+            max by (region, rhacs_cluster_name, rhacs_environment) (max_over_time(acs_probe_last_failure_timestamp[1w]))
+              >
+            max by (region, rhacs_cluster_name, rhacs_environment) (max_over_time(acs_probe_last_success_timestamp[1w]))
```

Comment:
1w sounds like quite long time. Why not 24h?

I think I need more explanation. I don't understand this query. We take max timestamp from last week and compare them.

Reply:
Note that we are dealing with timestamps here, which are monotonous. So taking the max represents the latest timestamp.

> I think I need more explanation. I don't understand this query. We take max timestamp from last week and compare them.

In planin English we are trying to do this: If either the last success or the last failure timestamp are not defined, use the most recent timestamp during the last week.

Why one week? It's arbitrary, but I thought it a reasonable compromise between going back to the end of time (which we logically would like), query performance and the probability that there is at least one undefined value in the period.